python-squid/src/test/java/org/sonar/plugins/python/api/cfg/ControlFlowGraphTest.java

```diff
@@ -404,6 +404,19 @@ public void class_def() {
     );
   }
 
+  /**
+   * The predecessors are constructed based on the successors, so we should not test them all the time
```

Comment:
'we' is used impersonally, do you prefer the passive voice?
In other CFG tests, we always test successors. Because predecessors are constructed based on successors, there is no need to add tests for predecessors in all other test cases, i.e. it won't increase the coverage.

Reply:
I updated the comment in 1c01d835e45b30377be88810ebf869c45c134685